cmd/bridge/main.go

```diff
@@ -204,3 +223,50 @@ func attachFileLogger(log logger.Logger, flagsConfig *relay.ContextFlagsConfig)
 
 	return fileLogging, nil
 }
+
+func buildNetMessenger(cfg relay.Config, marshalizer marshal.Marshalizer) (relayp2p.NetMessenger, error) {
+	nodeConfig := config.NodeConfig{
+		Port:                       cfg.P2P.Port,
+		Seed:                       cfg.P2P.Seed,
+		MaximumExpectedPeerCount:   0,
+		ThresholdMinConnectedPeers: 0,
+	}
+	peerDiscoveryConfig := config.KadDhtPeerDiscoveryConfig{
+		Enabled:                          true,
+		RefreshIntervalInSec:             5,
+		ProtocolID:                       cfg.P2P.ProtocolID,
+		InitialPeerList:                  cfg.P2P.InitialPeerList,
+		BucketSize:                       0,
+		RoutingTableRefreshIntervalInSec: 300,
+		Type:                             p2pPeerNetworkDiscoverer,
+	}
+
+	p2pConfig := config.P2PConfig{
+		Node:                nodeConfig,
+		KadDhtPeerDiscovery: peerDiscoveryConfig,
+		Sharding: config.ShardingConfig{
+			TargetPeerCount:         0,
+			MaxIntraShardValidators: 0,
+			MaxCrossShardValidators: 0,
+			MaxIntraShardObservers:  0,
+			MaxCrossShardObservers:  0,
+			Type:                    "NilListSharder",
```

Comment:
_maybe_ extract const for these magic numbers/values

Reply:
done